dev-util/roctracer/roctracer-5.0.2.ebuild

```diff
@@ -44,22 +46,29 @@ src_prepare() {
 	cmake_src_prepare
 
 	mv "${WORKDIR}"/rocprofiler-rocm-${PV} "${WORKDIR}"/rocprofiler || die
+	mv "${WORKDIR}"/hsa-class-*/test/util "${S}"/inc/ || die
+	rm "${S}"/inc/util/hsa* || die
+	cp -a "${S}"/src/util/hsa* "${S}"/inc/util/ || die
+
+	# change destination for headers to include/roctracer;
 
 	sed -e "/LIBRARY DESTINATION/s,lib,$(get_libdir)," \
-		-e "/add_subdirectory ( \${TEST_DIR} \${PROJECT_BINARY_DIR}/d" \
+		-e "/DESTINATION/s,\${DEST_NAME}/include,include/roctracer," \
 		-e "/install ( FILES \${PROJECT_BINARY_DIR}\/so/d" \
+		-e "/DESTINATION/s,\${DEST_NAME}/lib64,$(get_libdir),g" \
 		-i CMakeLists.txt || die
 
 	# do not download additional sources via git
 	sed -e "/execute_process ( COMMAND sh -xc \"if/d" \
 		-e "/add_subdirectory ( \${HSA_TEST_DIR} \${PROJECT_BINARY_DIR}/d" \
+		-e "/DESTINATION/s,\${DEST_NAME}/tool,$(get_libdir),g" \
 		-i test/CMakeLists.txt || die
 
 	hprefixify script/*.py
 }
 
 src_configure() {
-	export HIP_PATH="${EPREFIX}/usr"
+	export HIP_PATH="${EPREFIX}/usr/lib/hip"
```

Comment:
What's this?

Reply:
I should've have reviewed this diff.  Please squash and reorganize your commits into logical units.

Reply:
My mistake. I'll rebase it out.